cpp/mrc/include/mrc/core/utils.hpp

```diff
@@ -31,36 +31,6 @@ namespace mrc {
 
 std::uint16_t hash(const std::string&);
 
-template <typename T>
-constexpr auto type_name() noexcept
-{
-    std::string_view name = "[with T = <UnsupportedType>]";
-    std::string_view prefix;
```

Comment:
suffix and prefix should be moved to the _MSC_VER block as no other block needs them.